sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -207,6 +208,7 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
     // Returns true if the plan is supposed to be sorted.
     def isSorted(plan: LogicalPlan): Boolean = plan match {
       case _: Join | _: Aggregate | _: Generate | _: Sample | _: Distinct => false
+      case _: ShowColumnsCommand => true
```

Comment:
@viirya So it seemed odd to have the generated output files to have column names sorted which didn't reflect the columns. In the test case i had the table create like following.
```SQL
CREATE TABLE showcolumn2 (price int, qty int) partitioned by (year int, month int)
```
It seemed odd to me to have the generated output file report the columns as month, price, qty and year as opposed to price, qty, year and month.